packages/strapi-connector-mongoose/lib/buildQuery.js

```diff
@@ -508,6 +509,26 @@ const buildQuerySort = (model, filters) => {
   return [];
 };
 
+/**
+ * Add pagination operators for the aggregate
+ * @param {Object} model - Mongoose model
+ * @param {Object} filters - Filters object
+ */
+const buildQueryPagination = (model, filters) => {
+  const { limit, start } = filters;
+  const pagination = [];
+
+  if (start && start >= 0) {
+    pagination.push({ $skip: start });
+  }
+
+  if (limit && limit >= 0) {
```

Comment:
you might need to support limit = -1

Reply:
No need to support it here since -1 means no limit. (we already check for the validity of this parameter in convertRestQueryParams)